cmd/changelog-parser/testdata/expected_docs-release_output.txt

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<html xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd">
+  <head></head>
+  <body>
+    <h1>Release Notes</h1>
+    <p>The following features were introduced or enhanced in the unreleased Conjur OSS suite.</p>
```

Comment:
`unreleased`?

Reply:
`unreleased` is just the `version` variable that we use in that test: https://github.com/cyberark/conjur-oss-suite-release/blob/986cf2f0408de7298f1ffaa166b9b73181d12239/cmd/changelog-parser/main_test.go#L50